<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="attack_zone_list_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sknt_in_zone_label">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>6</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Список объектов в зоне действия SKNT-1</string>
    </property>
   </widget>
   <widget class="QListView" name="sucessed_list_view">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>311</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sucess_list_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Список успешно обработанных объектов</string>
    </property>
   </widget>
   <widget class="QListView" name="failed_list_view">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>311</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="failed_list_label">
    <property name="geometry">
     <rect>
      <x>684</x>
      <y>10</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Список объектов, покинувших действие SKNT-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="params_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>355</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Параметры SKNT-1:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ea_count">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="ea_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Орудия земля-воздух</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ee_count">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="ee_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Орудия земля-земля</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ew_count">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="ew_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Орудия земля-вода</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="processing_count">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="processing_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Скорость обработки объекта</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QLabel" name="damage_label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>350</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ущерб:</string>
    </property>
   </widget>
   <widget class="QLabel" name="prevented_damage_label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предотвращенный ущерб:</string>
    </property>
   </widget>
   <widget class="QLabel" name="prevented_damage_count_label">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>390</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="damage_count_label">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>420</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="damage_label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>420</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Понесенный ущерб:</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_object_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавление объекта</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="speed">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>390</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="type_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Тип</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Наименование</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Скорость</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Земля-воздух</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Земля-земля</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Земля-вода</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="cannons_count">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="cannons_coun_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Количество орудий для поражения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="obj_damage_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Наносимый ущерб</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="damage_count">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>560</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_data">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>attack_zone_list_view</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
